locales/de.json

```diff
@@ -47,7 +47,7 @@
 	"passenger-request-price.set_price": "In dieser App setzt der Fahrgast den Preis fest 😉 Gebe uns deinen Preis an (in lokaler Währung). Wie viel bist du bereit zu zahlen? (Oder gebe \"0\" (Null) an, wenn du es nicht weisst)",
 	"passenger-request-price.should_be_numeric": "Hoppla, der Preis sollte der numerische Wert (und in lokaler Währung) sein! Zum Beispiel: \"50\". Bitte versuche es erneut!",
 	"passenger-request-price.on_timeout": "Es scheint du wartest schon zu lange? Das tut uns leid. Wenn du keine Fahrt gefunden hast, empfehlen wir es später erneut zu versuchen.",
-	"passenger-contact-new-number.message": "🔔 {{driver}} akzeptiert deinen Preis! Entfernung zum Fahrer  ~{{distance}}. Rufe den Fahrer an um deinen Auftrag zu bestätigen:",
+	"passenger-contact-new-number.message": "🔔 {{driver}} hat deinen Preis akzeptiert! Entfernung zum Fahrer  ~{{distance}}. Rufe den Fahrer an um deinen Auftrag zu bestätigen:",
 	"driver-order-set-price.set_price": "Lege deinen Preis fest (in lokaler Währung). Du kannst Details einfügen, die deinen Preis erläutern. Zum Beispiel: \"20, wegen der Brückenreparatur\" oder \"15, wenn der Verkehr in Ordnung ist\"",
 	"passenger-contact-driver-price.message": "🔔 ⚠️ {{driver}} bietet einen anderen Preis: \"{{price}}\". Entfernung zum Fahrer ~{{distance}}. Wenn du mit dem Preis einverstanden bist, rufe den Fahrer an um deinen Auftrag zu bestätigen:",
 	"passenger-verify-location.verify_location": "Bitte schaue in die Karte oben (zum Vergrößern klicken), ist dieser Standort korrekt?",
```

Comment:
Right below here (in line 54): "deine Standort" -> "deinen Standort". Better yet, change whole sentence:

"Bitte versuche deine Standort zu senden durch Tippen auf die Büroklammer, und danach auf das Standort-Icon."
->
"Bitte tippe nun zuerst auf die Büroklammer und dann auf das Standort-Icon, um deinen Standort zu senden."

In line 55: "Haste du Bargeld" -> "Hast du Bargeld?"